examples/usb/src/main.xc

```diff
@@ -31,8 +24,8 @@ void endpoint0(chanend c_ep0_out, chanend c_ep0_in, client interface control i_c
   int handled;
   size_t len;
 
-  ep0_out = XUD_InitEp(c_ep0_out, XUD_EPTYPE_CTL | XUD_STATUS_ENABLE);
```

Comment:
could you explain why this was removed?

Reply:
That init function was part of lib_usb and several years old, when I moved to lib_xud, the init function has changed.